### ISSUE TITLE
Linked phone numbers

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#8063]
## User Story or Problem Statement
As a Veteran,  I want the phone numbers displayed in Facility Locator be linked so I can use my device to initiate the call.

Phone numbers are displayed in two places within Facility Locator:
- In search results
- On the Facility Detail page

![image](https://user-images.githubusercontent.com/55411834/79392710-158e6500-7f31-11ea-8091-648ca9625427.png)

![image](https://user-images.githubusercontent.com/55411834/79392979-90f01680-7f31-11ea-9c99-013ab43d64bd.png)

**Also:** notice in the Facility Detail page image that numbers are displayed twice: once as link for auto-dialer and one as routine display for "Main phone number".

**NOTE:** 
- Mental health phone numbers will be present for VA health facilities only. 
- In some cases, the Mental health phone number may be the same as the Main phone number. In this case, listing them twice with appropriate labels is expected. For example: 

> **Main Number:** 406-758-2700 
> **Mental Health:** 406-758-2700 

## Goal

- Users with capable devices can use click to call functionality. 
- Compliance with 508 standards
- Consistency across VA.gov

## Objectives or Key Results this is meant to further
-Objective 3: Deliver a usable and accessible experience

--

## Tasks
- [ ] Wrap **all** phone numbers in hyperlinks (on search results page as well as on Facility Detail page)
- [ ] Ensure both Main phone and Mental health phone numbers are converted. 
- [ ] Ensure that phone numbers are displayed only once (not once as auto-dialer link and once as routine number).
- [ ] Ensure unit tests are available.
- [ ] Add E2E tests to cover this particular functionality 
- [ ] Run Axe checks using the Chrome or Firefox browser plugin
- [ ] Test for color contrast and color blindness issues
- [ ] Zoom layouts to 400% at 1280px width
- [ ] Test with at least one screen reader
- [ ] Navigate using the keyboard only
- [ ] Check functionality across all facility types

> VA health
> VA benefit
> VA cemeteries
> Vet Centers
> Urgent care + VA urgent care
> Urgent care + Community urgent care (in VA's network)
> Community providers (in VA's network)

- [ ] Community urgent care pharmacies are currently suppressed in the VA facility drop-down but this functionality applies to these search results also.

## Acceptance Criteria
**For ALL facility and/or service types** 
- [ ] Unique phone numbers are displayed only once. 
- [ ] Any phone numbers (main and/or mental health) displayed on search result cards will have the visual appearance and functionality of a link. 
- [ ] Any phone numbers (main and/or mental health) displayed on facility detail pages will have the visual appearance and functionality of a link.
- [ ] Any phone numbers (main and/or mental health) displayed within Facility Locator will be accessible to users with assistive technology. 
- [ ] A Mental health phone number and a Main phone number is displayed whenever both are present in the data (even if the phone number itself is the same). 
- [ ] Unit tests pass
- [ ] End-to-end tests for new code shows 0 violations.
- [ ] Axe checks for this functionality pass
- [ ] Zoom testing passes
- [ ] Keyboard checks pass 
- [ ] Screenreader checks pass

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by mmiddaugh** [comment]
This issue should also rsolve [#6993]
